src/Microsoft.Framework.Runtime.Interfaces/IApplicationEnvironment.cs

```diff
@@ -11,6 +12,8 @@ public interface IApplicationEnvironment
         string Version { get; }
         string ApplicationBasePath { get; }
         string Configuration { get; }
+
+        [Obsolete("This method will be removed in a future version. Use IRuntimeEnvironment.Framework instead")]
```

Comment:
From the PR description:

> also moved the  RuntimeFramework  from  IApplicationEnvironment  to  IRuntimeEnvironment . However, I cannot remove the old one because xunit is using it. Therefore, I'll keep both for now, send a PR to xunit and then remove the old one. If I remove the old one, our build will fail.